src/barkhaPJun20/ArrayListFunctions_ReplaceWordFromArrayList.java

```diff
@@ -0,0 +1,56 @@
+/*Program 2 :
+Display on which index Techno is available, Replace all Techno with Credits from given ArrayList, and count total number of Credits.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+
+Output : 
+Techno word index in given arrayList is 1,3,5 
+["Maulik","Credits" ,"Krishna", "Credits", "Credits", "Credits"]
+Total Credits words :- 4*/
+
+package barkhaPJun20;
+
+import java.util.ArrayList;
+
+public class ArrayListFunctions_ReplaceWordFromArrayList {
+	int count=0;
+	
+	void replaceWordInArrayList(ArrayList<String> arrayList, String removeWord, String replaceWith) {
+		System.out.print("Index of "+removeWord+" in given ArrayList is:");
+		for (int index=0; index<arrayList.size(); index++) {
+			if (arrayList.get(index)==removeWord) {         
+				System.out.print(index+",");
+			}	
+		}
+		System.out.println();
+		System.out.println("-------------------------------------------------------------------------------------");
+		
+		System.out.println("ArrayList after replacing "+removeWord+" with "+replaceWith +" is:");
+			for (int index=0; index<arrayList.size(); index++) {
+				if (arrayList.get(index)==removeWord) {
+					arrayList.set(index, replaceWith);
+				}
+			}	
+		System.out.println(arrayList);	
+					
+		System.out.println("--------------------------------------------------------------------------------------");	
+		
+		for (int index=0; index<arrayList.size(); index++) {
+			if (arrayList.get(index)==replaceWith)
```

Comment:
Do not use == while comparing String == checks the reference not value, always check with .equals() method